### ISSUE TITLE
Missing 	# A list of entity types that should be excluded from activation range checks. 	excluded_entity_types = ["minecraft:hopper_minecart", "minecraft:warden", "minecraft:ghast", "minecolonies:citizen", "minecolonies:visitor"]

### ISSUE THREAD

**Issue by jmadarg** [Wesley1808/ServerCore#117]
**Describe the bug**
A clear description of what the bug is.

**Reproduce**

**Expected behavior**
A clear description of what you expected to happen.

**Versions** -Latest is not a version!
ServerCore:
Minecraft:

**Mod incompatibilities**
List any installed mods you think might play a role in causing this bug.

**Comment by jmadarg** [comment]
Oop, accidentally pressed enter, let me finish editing the issue

**Comment by Wesley1808** [comment]
I don't think this config setting existed in 1.3.5 either (I believe it was added in 1.3.7?), so makes sense if its not there.
1.3.6 for 1.19.2 isn't really a new version, its only some needed hotfixes on top of an old version of the mod. Though I prefer to drop support for 1.19.2 at this point it should be fairly trivial to add, so I might look into it once I have the time (and a working pc again)

**Comment by jmadarg** [comment]
Completely understandable! We previously used that option in 1.20.1 servers, I just kinda assumed it also existed on 1.19.2

I would really appreciate it if the option could be ported, this is an awesome mod! A must have!

**Comment by pwaksman973** [comment]
We should be able to blacklist the mod itself, instead of having to try to find out all the possible entity types for X mod, to make it work.

```yaml
inactive_tick_entity_blacklist:
  - "minecolonies:citizen"
  - "minecolonies:guard"
  - "minecolonies:visitor"
  - "minecolonies:mercenary"
  - "minecolonies:raider"
```

much better if we have something like:

```yaml
inactive_tick_entity_blacklist:
  - "minecolonies:*"
```

or 

```yaml
inactive_tick_mod_blacklist:
  - "minecolonies:*"
  - "minecoloni_addon"
```
